openfisca_web_api/loader/entities.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals, print_function, division, absolute_import
+from openfisca_core.commons import to_unicode
+
+
+def build_entities(tax_benefit_system):
+    entities = {
+        entity.key: build_entity(entity)
+        for entity in tax_benefit_system.entities
+        }
+    return entities
+
+
+def build_entity(entity):
+
+    entity_formated = {
+        'plural': entity.plural,
+        'description': to_unicode(entity.doc)
```

Comment:
#561 mentioned a `documentation` key containing the docstring. 
If we are to have a `description` key, it should probably contain `entity.label`